math/src/test/kotlin/ktx/math/rangesTest.kt

```diff
@@ -0,0 +1,244 @@
+package ktx.math
+
+import com.badlogic.gdx.math.MathUtils
+import com.badlogic.gdx.math.RandomXS128
+import org.junit.Assert.*
+import org.junit.Test
+import kotlin.math.abs
+
+/**
+ * Tests [ClosedRange]-related utilities.
+ */
+class RangesTest {
+
+  @Test
+  fun `should produce expected int range`() {
+    val center = -300
+    val tolerance = 10000
+    val range = center amid tolerance
+    assertEquals(range.first, center - tolerance)
+    assertEquals(range.last, center + tolerance)
+  }
+
+  @Test
+  fun `should multiply int range`() {
+    val original = -20..500
+    val multiplier = 37
+    val scaled = original * multiplier
+    val invertedScaled = multiplier * original
+    assertEquals(original.first * multiplier, scaled.first)
+    assertEquals(original.last * multiplier, scaled.last)
+    assertEquals(original.first * multiplier, invertedScaled.first)
+    assertEquals(original.last * multiplier, invertedScaled.last)
+  }
+
+  @Test
+  fun `should divide int range`() {
+    val original = -20..500
+    val divisor = 37
+    val divided = original / divisor
+    assertEquals(original.first / divisor, divided.first)
+    assertEquals(original.last / divisor, divided.last)
+  }
+
+  @Test
+  fun `should add to int range`() {
+    val original = -20..500
+    val addend = 37
+    val shifted = original + addend
+    assertEquals(original.first + addend, shifted.first)
+    assertEquals(original.last + addend, shifted.last)
+  }
+
+  @Test
+  fun `should subtract from int range`() {
+    val original = -20..500
+    val subtrahend = 37
+    val shifted = original - subtrahend
+    assertEquals(original.first - subtrahend, shifted.first)
+    assertEquals(original.last - subtrahend, shifted.last)
+  }
+
+  @Test
+  fun `should produce expected random int values`() {
```

Comment:
Note sure about this test. It seems to go into too much detail about how the internal implementation works. For tests such as this, I think I'd repeat the `range.random` call, save the results and check:
- If every value is within the range.
- If the values (roughly) produce uniform distribution.